server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -426,8 +426,11 @@ public class ShuffleServerConf extends RssBaseConf {
   public static final ConfigOption<ShuffleBufferType> SERVER_SHUFFLE_BUFFER_TYPE =
       ConfigOptions.key("rss.server.shuffleBuffer.type")
           .enumType(ShuffleBufferType.class)
-          .defaultValue(ShuffleBufferType.NONE)
-          .withDescription("The type for shuffle buffers.");
+          .defaultValue(ShuffleBufferType.LINKED_LIST)
+          .withDescription(
+              "The type for shuffle buffers. Setting as LINKED_LIST or SKIP_LIST."
+                  + "The default value is LINKED_LIST. SKIP_LIST will help to improve"
+                  + "the performance, but it don't support the show start feature of MR");
```

Comment:
In which scenarios can performance be improved? Or is it to say that performance can be improved in all scenarios except for MR with slow-start feature on? Could you please describe it in more detail?

it don't support the show start feature of MR
->
it doesn't support the slow-start feature of MR

Reply:
Updated, please take another look.